plugins/org.orbisgis.groovyeditor/test/org/orbisgis/ui/editors/groovy/GroovyCompletionProcessorTest.java

```diff
@@ -0,0 +1,100 @@
+/*
```

Comment:
Updates to this file should go in a dedicated PR as the current one is for the test on `ClassPathHandler`